src/dialog/llm/embeddings.py

```diff
@@ -26,7 +26,7 @@ def generate_embedding(document: str):
 def get_most_relevant_contents_from_message(message, top=5, dataset=None):
     message_embedding = generate_embedding(message)
     filters = [
-        CompanyContent.embedding.l2_distance(message_embedding) < 1,
+        CompanyContent.embedding.cosine_distance(message_embedding) < 0.2,
```

Comment:
Maybe this `0.2` should be configurable (later in a another PR maybe)? I have some real tests in a another project using cosine distance where the distances are above 0.2 but still they are relevant. This can happen due to many reasons, like big docs which will not achieve small distances to a small user query. The default I'd use would be as high as `0.9`.

Reply:
We can use a configuration variable with a default value, opening the possibility for the user to configure it in the best way for their use case.

**It would be here:**
https://github.com/talkdai/dialog/blob/a94146a79149e1f154f8f96919e5c0dd0e06f26d/src/dialog/settings.py#L18-L23